RocketApp/Infrastructure/Sources/NetworkClientExtensions/NetworkClient+.swift

```diff
@@ -0,0 +1,18 @@
+import Dependencies
+import Foundation
+import Networking
+import XCTestDynamicOverlay
+
+extension NetworkClient: DependencyKey {
+  public static var liveValue = NetworkClient(
```

Comment:
Zase to neni konzistentni, `RocketClient` to ma rozdeleny do vicero souboru, tady je to v jednom z nejakyho duvodu.

Reply:
Rozdeleno.